app-template/copay/sass/app.scss

```diff
@@ -0,0 +1,754 @@
+/***********************************************************************************************************
+ *
+ * Variables
+ * 
+ ***********************************************************************************************************/
+
+/*
+ * Define color palettes
+ */
+
+$color-palettes: (
+  copay-default: (
+    primary:        #192c3a,
+    secondary:      #31465b,
+    accent:         #1abb9b,
+    ctrl-primary:   #1abb9b,
+    ctrl-secondary: #0c5d4d,
+    ctrl-accent:    #192c3a,
+    text-light:     #c2c9d1,
+    text-dark:      #192c3a,
+    text-link:      #1abb9b,
+    success:        #1abb9b,
+    alert:          #bb331a
+  )
+);
+
+/*
+ * Choose color palette
+ */
+
+$selected-palette: copay-default;
+$palette: map-get($color-palettes, $selected-palette);
+
+/*
+ * Set colors based on the selected palette
+ */
+
+$app-primary-color:                map-get($palette, primary);
+$app-secondary-color:              map-get($palette, secondary);
+$app-accent-color:                 map-get($palette, accent);
+
+$ctrl-primary-color:               map-get($palette, ctrl-primary);
+$ctrl-secondary-color:             map-get($palette, ctrl-secondary);
+$ctrl-accent-color:                map-get($palette, ctrl-accent);
+
+$text-dark-color:                  map-get($palette, text-dark);
+$text-light-color:                 map-get($palette, text-light);
+$text-link-color:                  map-get($palette, text-link);
+
+$alert-color:                      map-get($palette, alert);
+$success-color:                    map-get($palette, success);
+
+/*
+ * Set element colors
+ */
+
+$icon-color:                       $app-accent-color;
+$icon-negative-color:              $app-accent-color;
+
+$primary-button-color:             $ctrl-primary-color;
+$primary-button-text-color:        $ctrl-accent-color;
+
+$secondary-button-color:           $ctrl-secondary-color;
+$secondary-button-text-color:      $ctrl-primary-color;
+
+$check-button-color:               $ctrl-primary-color;
+$toggle-button-color:              $ctrl-primary-color;
+
+$primary-background-color:         $app-primary-color;
+$secondary-background-color:       $app-secondary-color;
+$accent-background-color:          $app-accent-color;
+
+$gradient-bg-top-color:            $app-primary-color;
+$gradient-bg-bottom-color:         $app-secondary-color;
+
+/*
+ * Third-party brand specific colors
+ */
+
+$bitcoin-orange-color:                  #FF9900;
```

Comment:
These should probably be added to the rest of our variables: https://github.com/bitpay/copay/blob/master/src/sass/variables.scss